myapp/app/controllers/application_controller.rb

```diff
@@ -18,4 +19,12 @@ def record_not_found(e = nil)
     end
     redirect_to root_path, danger: t('error.messages.record_not_found')
   end
+
+  def maintenance_mode_on!
+    mainte_flg = Maintenance.exists?(status: 1)
+
+    return unless mainte_flg == true
```

Comment:
mainte_flgがBooleanであるのならこれで事足りるかと思います！

```suggestion
    return unless mainte_flg
```